src/lib/core/ripple/ripple-renderer.ts

```diff
@@ -17,6 +17,12 @@ export const RIPPLE_FADE_IN_DURATION = 450;
 /** Fade-out duration for the ripples in milliseconds. This can't be modified by the speedFactor. */
 export const RIPPLE_FADE_OUT_DURATION = 400;
 
+/**
+ * Timeout for ignoring mouse events. Mouse events will be temporary ignored after touch
+ * events to avoid synthetic mouse events.
+ */
+const ignoreMouseEventsTimeout = 800;
```

Comment:
This seems a little high? Wasn't the timeout 300ms?

Reply:
On Polymer it's 2000ms and in Materialize it's 500. On Material 1.x we used 650ms as a touch buffer for the interaction service. 800ms should be fine, and it doesn't really matter that much anyway.

Reply:
The const name here should be uppercased to match the others' naming.